cmd/cl001/ac001/explain/zz_generated.runner.go

```diff
@@ -50,7 +50,12 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		}
 	}
 
-	fmt.Println(strings.TrimSpace(e.Explain()))
+	s, err := e.Explain(ctx)
```

Comment:
I extended the interface to be a bit more future prove. We can generate most of the code but it is better to not cause too many disturbances down the road. So what we can anticipate can be prevented. Having the context given and being able to return errors should make it easier to implement different kinds of explainers.